packages/patternfly-4/react-charts/src/components/ChartBullet/utils/chart-bullet-size.ts

```diff
@@ -18,6 +18,35 @@ interface ChartBulletScaleInterface {
   width: number; // The chart width -- not SVG width
 }
 
+const scaleDimensions = ({
+  defaultSize,
+  height,
+  horizontal = true,
+  scale = 1,
+  value,
+  width
+}: ChartBulletScaleInterface) =>
+  horizontal
+    ? height > defaultSize
+      ? value + (height - defaultSize) * scale
+      : value - (defaultSize - height) * scale
+    : width > defaultSize
+    ? value + (width - defaultSize) * scale
+    : value - (defaultSize - width) * scale;
+
+// Scale bar width per the given size properties
+export const scaleBarWidth = (props: ChartBulletScaleInterface) => Math.max(scaleDimensions(props), 0);
+
+// Scale size per the given size properties
+export const scaleSize = ({ value, ...rest }: ChartBulletScaleInterface) =>
+  Math.round(
+    scaleDimensions({
```

Comment:
Is it necessary to refactor this?

Reply:
This was flagged by the [no-shadow](https://eslint.org/docs/rules/no-shadow) rule, as the function was named `scale` and created another variable inside it named `scale`.  It looks like the internal variable is pulled from props passed to this function, while the function itself is only called within the file and not exported, so changing the function name was easier than renaming the prop in this case.